buildSrc/src/main/java/Config.kt

```diff
@@ -19,14 +21,16 @@ object Config {
         val grettyVersion = "4.0.0"
         val gradleMavenPublishPlugin = "com.vanniktech:gradle-maven-publish-plugin:0.18.0"
         val dokkaPlugin = "org.jetbrains.dokka:dokka-gradle-plugin:$kotlinVersion"
+        val composeGradlePlugin = "org.jetbrains.compose:compose-gradle-plugin:$composeVersion"
     }
 
     object Android {
-        private val sdkVersion = 31
+        private val sdkVersion = 32
```

Comment:
We need this for compose latest version

Reply:
Is that stable already? if so, no problem.
If it's not yet, we can consider using 32 only for the compose packages.

Reply:
Yeah it is https://developer.android.com/studio/releases/platforms#12